src/pkg/services/m365/api/client.go

```diff
@@ -123,12 +123,18 @@ func newLargeItemService(
 	creds account.M365Config,
 	counter *count.Bus,
 ) (*graph.Service, error) {
-	a, err := NewService(creds, counter, graph.NoTimeout())
+	azureAuth, err := graph.NewAzureAuth(creds)
 	if err != nil {
-		return nil, clues.Wrap(err, "generating no-timeout graph adapter")
+		return nil, clues.Wrap(err, "generating azure authorizer")
 	}
 
-	return a, nil
+	a, err := NewService(
+		creds,
+		counter,
+		graph.NoTimeout(),
+		graph.AuthorizeRequester(azureAuth))
```

Comment:
Wondering why we need this change for the large item. This is a graph client so it should already have auth built in.

Reply:
@ryanfkeepers Could you please address this comment? Thanks!

Reply:
Just noticed that this change was removed in the next PR in the chain. https://github.com/alcionai/corso/pull/5200/files#diff-ea424cd36ba63e8261c19fa218f00c5e42bc30e874b2dd5a5eb09d8feb20574bR131 

So we are good.